source/backend/dal/Repositories/PropertyRepository.cs

```diff
@@ -395,6 +395,7 @@ public PimsProperty TransferFileProperty(PimsProperty property, bool isOwned, bo
 
             existingProperty.IsPropertyOfInterest = isPropertyOfInterest;
             existingProperty.IsOwned = isOwned;
+            existingProperty.IsDisposed = isDisposed;
```

Comment:
Besides from the Disposed Flag, shouldn't the PropertyClassificationTypeCode be updated to "DISPOSED" also?

Reply:
We don't actually use the PropertyClassificationTypeCode, it is a holdover from CITZ pims.

Reply:
Could we consider the idea to have a new method called "DisposeProperty"?